server/controllers/routes/user/business.js

```diff
@@ -0,0 +1,76 @@
+const { compare } = require('bcryptjs');
+
+const { getPassword } = require('./../../../database/queries/getPassword');
+const { businessDataSchema } = require('../../utils/businessSchema');
+const { updateBusinessDataQuery } = require('./../../../database/queries/updateBusinessData');
+
+exports.updateBusiness = (req, res, next) => {
+  const {
+    oldPassword,
+    organisationName,
+    type,
+    website,
+    city,
+    country,
+    address,
+    zipCode,
+    facebook,
+    twitter,
+    instagram,
+  } = req.body;
+  const { id } = req.user;
+
+  getPassword(id)
+    .then(dbRes => compare(oldPassword, dbRes.rows[0].password))
+    .then((passMatch) => {
+      if (passMatch) {
+        return businessDataSchema.isValid({
+          organisationName,
+          type,
+          website,
+          city,
+          country,
+          address,
+          zipCode,
+          facebook,
+          twitter,
+          instagram,
+        });
+      }
+      const objError = new Error('Retry, password is wrong');
+      objError.statusCode = 401;
+      throw objError;
+    })
+    .then((valid) => {
+      if (valid) {
+        return updateBusinessDataQuery(organisationName,
+          type,
+          website,
+          city,
+          country,
+          address,
+          zipCode,
+          facebook,
+          twitter,
+          instagram,
+          id);
+      }
+      const objError = new Error('Bad Request');
+      objError.statusCode = 400;
+      throw objError;
+    })
+    .then(() => res.send({
+      data: 'Business Data Updated Successfully',
```

Comment:
Return the object of the updated data

Reply:
@IsraaSulaiman, Done
Here: [send updated data in response](https://github.com/FACG6/social-streets/pull/141/commits/5f0699aa2cdd53d4889636362479329b1ecab9ea)